subprojects/test-runner/service/src/testFixtures/kotlin/com/avito/runner/service/worker/listener/FakeMethodsInvocationDeviceListener.kt

```diff
@@ -0,0 +1,51 @@
+package com.avito.runner.service.worker.listener
+
+import com.avito.runner.service.model.DeviceTestCaseRun
+import com.avito.runner.service.model.intention.Intention
+import com.avito.runner.service.model.intention.State
+import com.avito.runner.service.worker.device.Device
+import com.avito.runner.service.worker.model.DeviceInstallation
+
+class FakeMethodsInvocationDeviceListener : StubDeviceListener() {
```

Comment:
Stub extending another stub looks excessive.
If I'm not mistaken, the sense of manual stubs is only to avoid fragile Mockito.
This means that recording invocations can be the responsibility of `StubDeviceListener` itself.